src/openclon12/device.cpp

```diff
@@ -599,14 +600,9 @@ void Device::FlushAllDevices(TaskPoolLock const& Lock)
     }
 }
 
-//std::unique_ptr<D3D12TranslationLayer::PipelineState> D3DDevice::CreatePSO(D3D12TranslationLayer::COMPUTE_PIPELINE_STATE_DESC const& Desc)
-//{
-//    std::lock_guard PSOCreateLock(m_PSOCreateLock);
-//    return std::make_unique<D3D12TranslationLayer::PipelineState>(&ImmCtx(), Desc);
-//}
-
-void D3DDevice::ExecuteTasks(Submission& tasks)
+void D3DDevice::ExecuteTasks(std::unique_ptr<Submission> spTasks)

```

Comment:
What does the `s` prefix stand for here? At first I thought it was a typo, but then saw that you have it in a couple places so probably intentional. The ones that come to mind don't seem to fit here.

Reply:
Heh, it's a bit of a remnant from when I cared more about Hungarian notation I guess. `p` for pointer, `sp` for smart pointer.